packages/keyring-controller/package.json

```diff
@@ -45,9 +47,9 @@
     "@ethereumjs/common": "^3.2.0",
     "@ethereumjs/tx": "^4.2.0",
     "@keystonehq/bc-ur-registry-eth": "^0.9.0",
+    "@lavamoat/allow-scripts": "^2.3.1",
     "@metamask/auto-changelog": "^3.4.4",
     "@metamask/eth-sig-util": "^7.0.1",
-    "@metamask/preferences-controller": "^6.0.0",
```

Comment:
Hey @danroc, why was this moved from dev dependenices to dependencies? I'm asking because this conflicts with one of my PRs, where this is being removed as a dependency completely.